pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -300,10 +329,10 @@
   heading_year_long_intro_course: "Year-long introductory course"
   heading_flexible_units: "Flexible units"
   heading_collections: "Collections"
+  heading_find_a_module: "Find a module for your classroom"
 
-  professional_learning_section_subhead: "Get the support you need as you prepare to teach."
-  professional_learning_section_desc_v1: "Compared with similarly-situated schools, [a school's participation in the Code.org Professional Learning program causes an estimated five-fold increase](%{url}) in the number of students that take, and earn qualifying scores on the AP Computer Science Principles exam. And teachers love it! Over 90% rank it the best professional development ever."
-  professional_learning_section_block_facilitator_led_workshops: "Our highly supportive Professional Learning Program will bring you together with a group of teachers to learn from each other as well as expert facilitators as you take on the first year of teaching the free CS Principles curriculum to your students. In-person and virtual workshops starting this summer."
+  professional_learning_section_desc: "Get the support you need as you prepare to teach. Teachers love it, with over 90% ranking it the best professional development ever!"
+  professional_learning_section_block_facilitator_led_workshops: "Our highly supportive Professional Learning Program will bring you together with a group of teachers to learn from each other as well as expert facilitators as you take on the first year of teaching the free curriculum to your students. In-person and virtual workshops starting this summer."
```

Comment:
Nit: not a big deal but I'm wondering if the last sentence would read a little better as "In-person and virtual workshops are starting this summer."